Bottom: e5a2b0024cb142389ac00cc7ac18dc9614b47b33
Top:    96d6facc0af645b59ca969b351bf842ada25e744
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-25 15:44:01 +0800

Refresh of tuntap-xdp-transmission

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 2808029..19906c8 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -656,8 +656,9 @@ static void tun_ptr_free(void *ptr)
 		struct xdp_buff *xdp = tun_ptr_to_xdp(ptr);
 
 		put_page(virt_to_head_page(xdp->data));
-	} else
+	} else {
 		__skb_array_destroy_skb(ptr);
+	}
 }
 
 static void tun_queue_purge(struct tun_file *tfile)
@@ -2302,8 +2303,9 @@ static int tun_ptr_peek_len(void *ptr)
 			return xdp->data_end - xdp->data;
 		}
 		return __skb_array_len_with_tag(ptr);
-	} else
+	} else {
 		return 0;
+	}
 }
 
 static int tun_peek_len(struct socket *sock)
